axelrod/strategies/cycler.py

```diff
@@ -1,13 +1,15 @@
-from axelrod.actions import Actions, Action
+from axelrod.actions import Actions, Action, str_to_actions
 from axelrod.player import Player
-
+import itertools
 import copy
 
+C, D = Actions.C, Actions.D
+
 
 class AntiCycler(Player):
     """
     A player that follows a sequence of plays that contains no cycles:
-    C CD CCD CCCD CCCCD CCCCCD ...
+    CDD  CD  CCD CCCD CCCCD ...
```

Comment:
Was this wrong originally? It's supposed to be n-Cs and 1-D for n in 0, 1, 2, ...